tutorials/accelerated-python/notebooks/libraries/20__cudf__nyc_parking_violations.ipynb

```diff
@@ -1,2436 +1,667 @@
-   "source": [
-    "* `cudf.pandas` docs: https://docs.rapids.ai/api/cudf/stable/cudf_pandas/\n",
-    "* cuDF documentation: https://docs.rapids.ai/api/cudf/stable\n",
-    "* cuDF API reference: https://docs.rapids.ai/api/cudf/stable/user_guide/api_docs/"
-   ]
-  }
- ],
- "metadata": {
-  "colab": {
-   "gpuType": "T4",
-   "provenance": []
-  },
-  "kernelspec": {
-   "display_name": "Python 3 (RAPIDS 25.10)",
-   "language": "python",
-   "name": "cudf-cu12-25.10"
-  },
-  "language_info": {
-   "codemirror_mode": {
-    "name": "ipython",
-    "version": 3
-   },
-   "file_extension": ".py",
-   "mimetype": "text/x-python",
-   "name": "python",
-   "nbconvert_exporter": "python",
-   "pygments_lexer": "ipython3",
-   "version": "3.13.5"
-  }
- },
- "nbformat": 4,
- "nbformat_minor": 5
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "# DataFrames with Pandas and cuDF"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Table of Contents\n",
+        "\n",
+        "1. [Introduction](#1.-Introduction)\n",
+        "2. [Pandas Essentials](#2.-Pandas-Essentials)\n",
+        "   - [2.1 Series and DataFrame Objects](#2.1-Series-and-DataFrame-Objects)\n",
+        "   - [2.2 Selecting and Filtering Data](#2.2-Selecting-and-Filtering-Data)\n",
+        "   - [2.3 Sorting](#2.3-Sorting)\n",
+        "   - [2.4 Summarizing Data](#2.4-Summarizing-Data)\n",
+        "   - [2.5 Grouped Aggregations (groupby)](#2.5-Grouped-Aggregations-(groupby))\n",
+        "   - [2.6 String Operations](#2.6-String-Operations)\n",
+        "   - [2.7 Time Series](#2.7-Time-Series)\n",
+        "   - [2.8 User-Defined Operations (apply)](#2.8-User-Defined-Operations-(apply))\n",
+        "3. [Enter cuDF: GPU DataFrames](#3.-Enter-cuDF:-GPU-DataFrames)\n",
+        "   - [3.1 Exercise: Date Formatting Failure](#3.1-Exercise:-Date-Formatting-Failure)\n",
+        "   - [3.2 Exercise: Why `.apply()` Breaks Down in cuDF](#3.2-Exercise:-Why-`.apply()`-Breaks-Down-in-cuDF)\n",
+        "4. [Exercise: Analyzing Real Data (NYC Parking Violations)](#4.-Exercise:-Analyzing-Real-Data-(NYC-Parking-Violations))\n",
+        "   - [Step 0: Download Data](#Step-0:-Download-Data)\n",
+        "   - [Task 1: Data Inspection (Pandas)](#📝-Task-1:-Data-Inspection-(Pandas))\n",
+        "   - [Task 2: Analyze Taxis (Pandas)](#📝-Task-2:-Analyze-Taxis-(Pandas))\n",
+        "   - [Task 3: GPU Acceleration (cuDF)](#📝-Task-3:-GPU-Acceleration-(cuDF))\n",
+        "5. [Conclusion](#Conclusion)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## 1. Introduction\n",
+        "\n",
+        "In this notebook, we will build a foundation in data manipulation using **Pandas**, the industry standard for Python data analysis. Then, we will transition to **cuDF**, which allows us to run standard Pandas-like code on the GPU.\n"
```

Comment:
I would keep the tone here a bit more neutral. Instead of `industry standard`, let's just call it `a popular tool`.